frameworks/g-cloud-9/questions/services/standardsCSASTARExclusions.yml

```diff
@@ -0,0 +1,20 @@
+name: What the CSA STAR doesn’t cover
```

Comment:
I'm curious about why Github doesn't do syntax highlighting for this and the following `standards...` questions.